pkgs/top-level/all-packages.nix

```diff
@@ -1102,6 +1102,8 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) Cocoa CoreGraphics Foundation IOKit Kernel OpenGL;
   };
 
+  kitty-themes  = callPackage ../applications/terminal-emulators/kitty-themes  {};
```

Comment:
I think `terminal-emulators` is not the right category here since this is not a terminal emulator but a collection of themes. `/pkgs/data/themes/` seems to also not fit because the `coding-conventions` document says to use that directory "If it’s a theme for a desktop environment, a  window manager or a display manager", which is not the case for kitty-themes. So maybe just put it in `/pkgs/misc/`?